client/src/unifyfs-sysio.c

```diff
@@ -527,6 +527,76 @@ int UNIFYFS_WRAP(__fxstat)(int vers, int fd, struct stat* buf)
 }
 #endif
 
+
+#ifdef HAVE_SYS_STATFS_H
+
+/* tmpfs seems like a safe choice for something like UnifyFS */
+#ifndef TMPFS_MAGIC
+#define TMPFS_MAGIC 0x01021994
+#endif
```

Comment:
I think we're cool enough to have our own f_type :sunglasses:      Maybe?:
```
0x554E4946  
   U N I F
```

Reply:
I'll leave that decision up to the people with the money.